components/LandingPage.tsx

```
@@ -0,0 +1,331 @@
+              <button className="text-neutral-800 px-8 py-4 rounded-lg font-semibold hover:bg-neutral-100 transition-colors">
+                See how it works
+              </button>
+            </div>
+          </div>
+
+          {/* App Screenshot */}
+          <div className="relative max-w-5xl mx-auto">
+            <div className="bg-white rounded-2xl shadow-2xl border border-neutral-200 overflow-hidden">
+              <div className="bg-neutral-100 px-6 py-4 border-b border-neutral-200">
+                <div className="flex items-center space-x-2">
+                  <div className="w-3 h-3 bg-red-400 rounded-full"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)